halo2_gadgets/src/utilities/lookup_range_check.rs

```diff
@@ -877,7 +1024,7 @@ mod tests {
             mut layouter: impl Layouter<F>,
         ) -> Result<(), Error> {
             // Load table_idx
-            config.load(&mut layouter)?;
+            config.load_only_range_check_table(&mut layouter)?;
```

Comment:
`only` is confusing long term. maybe just `load_range_check_table()`

Reply:
Done